server/app/routes/index.js

```diff
@@ -4,7 +4,8 @@ module.exports = router;
 
 router.use('/members', require('./members'));
 
-router.use('/api', require('./api'))
+router.use('/', require('./api'));
+
```

Comment:
realized that if we put '/api' it was using '/api/api/..." because '/api' points to './routes' in top-level  index